src/main/java/jalau/at18/kingoftokyo/DiceIndentifier.java

```diff
@@ -0,0 +1,23 @@
+package jalau.at18.kingoftokyo;
+
+public class DiceIndentifier {
+    private DiceRoller diceRoller;
+
+    public DiceIndentifier() {
+        this.diceRoller = new DiceRoller();
+    }
+
+    public String[] value() {
+        String[] numberRandom = diceRoller.showListDices().split("");
+        return numberRandom;
+    }
+
+    public boolean identifyDices() {
```

Comment:
this method does not seem to be ready to be included yet, can we remove it for now? once it has actual implementation , can be pushed as part of a next PR